framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -1003,6 +1003,9 @@ protected boolean visitWildcardSupertype(
 
     protected boolean visitWildcardSubtype(
             AnnotatedWildcardType subtype, AnnotatedTypeMirror supertype, VisitHistory visited) {
+        if (subtype.isUninferredTypeArgument()) {
+            return ignoreUninferredTypeArguments;
```

Comment:
In BaseTypeVisitor, should we add a warning when type arguments inference fails?  The warning would be issued even if `-AconservativeUninferredTypeArguments` wasn't passed.